File: spring-s-m-3/s-m-common/src/main/java/pres/qianmuna/ssm/model/UserModel.java
package pres.qianmuna.ssm.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

import java.io.Serializable;

/**
 * @author HJC
 * @version 1.0
 * 谦谦君子 卑以自牧也
 * @date 2020/6/10  16:50
 * @description :
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@Accessors( chain = true)
public class UserModel implements Serializable {

    private static final long serialVersionUID = -5420716201302024404L;

    private Long uid;
    private String userName;
    private String password;
    private String updateDate;



}

File: kotlin-mybatis/src/main/kotlin/pres/qianmuna/kt/entity/UserModel.kt
package pres.qianmuna.kt.entity

/**
@author HJC
@date 2020/6/4  17:20
@description :
@version 1.0
谦谦君子 卑以自牧也
 */
data class UserModel(private val id:Long? = null,
                     private val userCode: String? = null,
                     private val userName: String? = null,
                     private val userPassword: String? = null,
                     private val gender: Int? = null,
                     private val birthday: String? = null,
                     private val phone: String? = null,
                     private val address: String? = null,
                     private val userRole: Long? = null,
                     private val createdBy: Long? = null ,
                     private val creationDate: String? = null,
                     private val modifyBy: Long? = null,
                     private val modifyDate: String? = null,
                     private val idPicPath: String? = null,
                     private val workPicPath: String?= null ,
                     private val addressModel: MutableList<AddressModel>? = null){

    override fun toString(): String {
        return "UserModel(id=$id, userCode=$userCode, userName=$userName, userPassword=$userPassword, gender=$gender, birthday=$birthday, phone=$phone, address=$address, userRole=$userRole, createdBy=$createdBy, creationDate=$creationDate, modifyBy=$modifyBy, modifyDate=$modifyDate, idPicPath=$idPicPath, workPicPath=$workPicPath, addressModel=$addressModel)"
    }

}